scss/layouts/_footer.scss

```
@@ -0,0 +1,213 @@
+@import '../base/mixins';
+@import '../base/variables';
+.footer {
+  background-color: $dark;
+  color: #b7b7b7;
+  font-family: $secondary-font;
+  font-weight: 500;
+
+  &__head {
+    font-size: 1.4rem;
+    @include gridTemplate($column: 6, $gap: 4rem);
+    padding: 3.5rem 0;
+    h3 {
+      line-height: 2.6rem;
+    }
+    border-bottom: 1px solid $dark-lighter;
+  }
+  &__social {
+    grid-column: 1 / span 2;
+    h3 {
+      line-height: 2rem;
+      font-size: 1.4rem;
+      padding-bottom: 1rem;
+    }
+  }
+
+  &__icon {
+    span {
+      padding-right: 2rem;
+
+      @include hoverTransition($white);
+    }
+  }
+  &__email {
+    h3 {
+      font-size: 1.4rem;
+    }
+  }
+  &__form {
```

Mỗi block cách 1 dòng nhé
```suggestion
  }
  
  &__email {
    h3 {
      font-size: 1.4rem;
    }
  }
  
  &__form {
```